CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ find_package(catkin REQUIRED
     moveit_ros_planning_interface
     pluginlib
     geometric_shapes
+    moveit_visual_tools
```

Comment:
I have always used deprecated warnings within a ROS release, but have sometimes changed API a little for each new ROS version. Now that my PhD is rounding out development will likely slow down and keep it very stable.

Assuming it is wanted, I've decided yes I'm happy to move it to ros-planning. I'll open a issue for it to discuss further. What are your thoughts on the dependent package rviz_visual_tools?

Is this the last issue blocking this PR from being merged?